prime-router/src/main/kotlin/history/SubmissionHistory.kt

```diff
@@ -303,17 +314,73 @@ class DetailedSubmissionHistory(
     }
 
     /**
-     * Enrich a parent detailed history with details from the process or translate action.
+     * Enrich a parent detailed history with details from the route and translate actions.
+     * Add destinations, errors, and warnings, to the history details.
+     * Note: Route/Translate is exclusive to the Universal pipeline
+     * See enrichWithProcessAction for the COVID pipeline counterpart
+     *
+     * @param descendants[] route and translate actions that will be used to enrich
+     */
+    private fun enrichWithRoutingAndTranslationActions(descendants: List<DetailedSubmissionHistory>) {
+        require(topic == Topic.FULL_ELR.json_val) {
+            "Route and Translate enrichment is only available for the Universal Pipeline"
+        }
+
+        val filterLogs = mutableListOf<DetailedActionLog>()
+
+        // Grab the filter logs generated during the "route" action, as well as errors and warnings
+        descendants.filter { it.actionName == TaskAction.route }.forEach { descendant ->
+            filterLogs += descendant.logs.filter { log -> log.type == ActionLogLevel.filter }
+            errors += descendant.errors
+            warnings += descendant.warnings
+        }
+
+        // Grab destinations from the "translate" action, if the submission made it that far
+        descendants.filter { it.actionName == TaskAction.translate }.forEach { descendant  ->
```

Comment:
The linter doesn't like this line either, you have an extra " " between `descendant` and `->`,  try to fix that before merging as well!

Reply:
Yup, forgot to run lint. Reran and it's fixed now